### PR TITLE
[TTAHUB-799] Upgrade Font Awesome to Version 6 Pro

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

Upgrade Font Awesome to Version 6 Pro

I have also installed the optional versions of the icons (solid, outline, etc) and kept the free versions.

## How to test

Visit any pages in the app that use Font Awesome Icon. These icons should show as they did before.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-799

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/components/GoalsTable/__tests__/GoalsTable.js [hunk 1]
```diff
@@ -391,8 +391,8 @@ describe('Goals Table', () => {
       await screen.findByText('TTA goals and objectives');
 
       expect(screen.getAllByRole('cell')[0]).toHaveTextContent(/needs status/i);
-
-      expect(document.querySelector('.fa-exclamation-circle')).toHaveAttribute(
+      screen.debug(undefined, 100000);
```
Comment (jasalisbury) [inline frontend/src/components/GoalsTable/__tests__/GoalsTable.js:394]:
```suggestion
```

**Review by jasalisbury — APPROVED** [review]
Did `FONTAWESOME_NPM_AUTH_TOKEN` get added for all envs in circleci?

File: frontend/src/components/GoalsTable/__tests__/GoalsTable.js (hunk 1 above)
Comment (AdamAdHocTeam) [reply frontend/src/components/GoalsTable/__tests__/GoalsTable.js]: I removed the debug. 

It seems the current Cicle CI env var is for "Head-Start-TTADP" project.

Comment (AdamAdHocTeam) [reply frontend/src/components/GoalsTable/__tests__/GoalsTable.js]: The new FA has also been deployed to DEV works as expected.

**Comment by thewatermethod** [comment]
Screen Shot 2022-05-02 at 9 40 52 AM
This is the old sidebar icon

Screen Shot 2022-05-02 at 9 40 45 AM
This is what's on dev

Everything else LGTM

**Comment by AdamAdHocTeam** [comment]
> Screen Shot 2022-05-02 at 9 40 52 AM This is the old sidebar icon
> 
> Screen Shot 2022-05-02 at 9 40 45 AM This is what's on dev
> 
> Everything else LGTM

I have updated the AR icon to be the vertical bar chart.

**Pull request merged**